src/Core/Settings.h

```diff
@@ -77,7 +77,7 @@ class IColumn;
     M(UInt64, s3_max_connections, 1024, "The maximum number of connections per server.", 0) \
     M(UInt64, hsts_max_age, 0, "Expired time for hsts. 0 means disable HSTS.", 0) \
     M(Bool, extremes, false, "Calculate minimums and maximums of the result columns. They can be output in JSON-formats.", IMPORTANT) \
-    M(Bool, use_uncompressed_cache, false, "Whether to use the cache of uncompressed blocks.", 0) \
+    M(Bool, use_uncompressed_cache, true, "Whether to use the cache of uncompressed blocks.", 0) \
```

Comment:
Temporarly enable this setting, because we have only 2 tests with `use_uncompressed_cache=true`.